client/src/util/stateManager/schemaHelpers.js

```diff
@@ -30,6 +31,15 @@ export function indexEntireSchema(schema) {
 	return schema;
 }
 
+export function sortAllCategorical(schema) {
+	/* UI relies on OBS annotation categories being in presentation sort order */
+	schema.annotations.obs.columns.forEach(c => {
+		if (c.categories) {
+			c.categories = catLabelSort(c.writable, c.categories);
```

Comment:
Spacing looks off here

Reply:
the whole file was tab indented.  Fixed now.